modules/lvms-scaling-storage-expand-pvc.adoc

```diff
@@ -4,22 +4,38 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="lvms-scaling-expand-pvc_{context}"]
-= Expanding PVCs
+= Expanding a persistent volume claim
 
-To leverage the new storage after adding additional capacity, you can expand existing persistent volume claims (PVCs) with LVM Storage.
+After scaling up the storage of a cluster, you can expand the existing persistent volume claims (PVCs). 
+
+To expand a PVC, you must update the `requests.storage` field in the PVC.
 
 .Prerequisites
 
 * Dynamic provisioning is used.
-* The controlling `StorageClass` object has `allowVolumeExpansion` set to `true`.
+* The `StorageClass` object associated with the PVC has the `allowVolumeExpansion` field set to `true`.
 
 .Procedure
 
-. Modify the `.spec.resources.requests.storage` field in the desired PVC resource to the new size by running the following command:
+. Log in to the OpenShift CLI (`oc`).
+
+. Update the value of the `spec.resources.requests.storage` field to a value that is greater than the current value by running the following command:
 +
 [source,terminal]
 ----
-oc patch <pvc_name> -n <application_namespace> -p '{ "spec": { "resources": { "requests": { "storage": "<desired_size>" }}}}'
+$ oc patch pvc <pvc_name> -n <application_namespace> -p \ <1>
```

Comment:
This will fix the syntax highlighting in the output:
```suggestion
$ oc patch pvc <pvc_name> -n <application_namespace> -p \// <1>
```

Reply:
Updated as per the suggestion.